src/controlplane/controlplane_test.go

```diff
@@ -109,7 +109,13 @@ func Test_Scraper_Autodiscover_all_cp_components(t *testing.T) {
 
 			// apiserverStorageObjects replaces etcObjectCounts in k8s versions above 1.23
 			if testutil.IsBelow(version, testutil.Testdata123) {
-				versionAsserter = versionAsserter.Excluding(exclude.Metrics("apiserverStorageObjects"))
```

Comment:
i'm wondering why this test didn't catch the metric name "issue" we found with the e2e for some metrics like the `apiserver_current_inflight_requests`